hack/ginkgo-e2e.sh

```diff
@@ -99,5 +99,6 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --num-nodes="${NUM_MINIONS:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  ${E2E_MIN_STARTUP_PODS+"--minStartupPods=${E2E_MIN_STARTUP_PODS}"} \
```

Comment:
yeah, I just copied the line below this which seems to DTRT. (I also tested it locally.)